src/Layout/AppLayout.jsx

```diff
@@ -229,6 +235,7 @@ AppLayout.propTypes = {
   footerContent: PropTypes.node,
   footerTooltip: PropTypes.node,
   footerBgImage: PropTypes.node,
+  headerPadding: PropTypes.oneOfType(PropTypes.string, PropTypes.array),
```

Comment:
Can be also a number?

```suggestion
  headerPadding: PropTypes.oneOfType(PropTypes.string, PropTypes.array, PropTypes.number),
```

Reply:
✅